osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/application/services/DeviceManagementService.java

```diff
@@ -143,22 +143,24 @@ public void sendMeasurements(final String deviceIdentification, final GetDataRes
         // Correlation ID is generated @ WS adapter, domain+version is
         // hard-coded
         // for now
-        final ProtocolResponseMessage responseMessage = new ProtocolResponseMessage.Builder().dataObject(response)
-                .deviceMessageMetadata(new DeviceMessageMetadata(deviceIdentification, "no-organisation",
-                        "no-correlationUid", DeviceFunctionDto.GET_DATA.name(), 0))
-                .result(ResponseMessageResultType.OK).domain("MICROGRIDS").domainVersion("1.0").build();
+        final ProtocolResponseMessage responseMessage = new ProtocolResponseMessage.Builder().dataObject(
+                response).deviceMessageMetadata(
+                new DeviceMessageMetadata(deviceIdentification, "no-organisation", "no-correlationUid",
+                        MessageType.GET_DATA.name(), 0)).result(ResponseMessageResultType.OK).domain(
+                "MICROGRIDS").domainVersion("1.0").build();
         this.responseSender.send(responseMessage);
     }
 
     public void sendPqValues(final String deviceIdentification, final String reportDataSet,
             final GetPQValuesResponseDto response) throws ProtocolAdapterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.adapter.protocol.iec61850.exceptions.ProtocolAdapterException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)